SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

```diff
@@ -21,6 +21,8 @@ class USpatialNetDriver;
 class USpatialReceiver;
 class USpatialStaticComponentView;
 
+DECLARE_DELEGATE_OneParam(CreateEntityDelegate, const Worker_CreateEntityResponseOp&);
```

Comment:
Unsure this class the best place to be defining the delegate / callback methods vs. something like the netdriver since it's higher level functionality in ensuring the interest is correct compared to dispatcher callbacks which is what the dispatcher does otherwise currently. 

Don't feel too strongly but see what others in GDK say

Reply:
@m-samiec told me to place it here for now.